tests/foreman/smoke/test_smoke.py

```diff
@@ -39,3 +42,93 @@ def test_selinux_foreman_module(self):
             rpm_version = rpm_version[:-2]
 
         self.assertEqual(rpm_version.replace('-', '.'), semodule_version)
+
+    def test_installer_check_services(self):
+        """@Test: Services services start correctly
+
+        @Feature: Installer
+
+        @Assert: All services {katello-jobs, tomcat6, foreman, pulp,
+        passenger-analytics, httpd, foreman_proxy, elasticsearch, postgresql,
+        mongod} are started
+
+        """
+        services = (
+            'elasticsearch',
+            'foreman',
+            'foreman-proxy',
+            'foreman-tasks',
+            'httpd',
+            'mongod',
+            'postgresql',
+            'pulp_celerybeat',
+            'pulp_resource_manager',
+            'pulp_workers',
+            'qdrouterd',
+            'qpidd',
+            'tomcat',
+        )
+
+        # check `services` status using service command
+        if get_host_info()[1] >= 7:
+            status_format = 'systemctl status {0}'
+        else:
+            status_format = 'service {0} status'
+
+        for service in services:
+            result = ssh.command(status_format.format(service))
+            if service == 'foreman':
+                self.assertEqual(result.return_code, 3)
```

Comment:
Lets remove the foreman process if it not necessary, we can add it back later if needed.